pandas/tests/frame/methods/test_to_dict_of_blocks.py

```diff
@@ -56,11 +56,16 @@ def test_to_dict_of_blocks_item_cache():
 
     df._to_dict_of_blocks()
 
-    # Check that the to_dict_of_blocks didn't break link between ser and df
-    ser.values[0] = "foo"
-    assert df.loc[0, "b"] == "foo"
+    if using_copy_on_write:
+        # TODO(CoW) we should disallow this, so `df` doesn't get updated
```

Comment:
Maybe worth xfailing this branch? On first read I ignored the comment and couldn't understand why this was branching